Monika After Story/game/script-farewells.rpy

```diff
@@ -947,6 +947,25 @@ label bye_going_somewhere:
         $ _history_list.pop()
         m 1lksdlc "..."
         m 1eksdlb "What am I saying? Of course I'll go with you!"
+        if mas_isMoniLove(higher=True):
```

Comment:
You have this check inside a `mas_isMoniDis(lower=True):`, so this will never execute.